ffi/input_android.lua

```diff
@@ -167,6 +180,10 @@ function input.waitForEvent(usecs)
                 android.LOGI("Engine thread destroy requested!")
                 return
             end
+        elseif poll_state == C.ALOOPER_POLL_WAKE then
+            -- this happens, when ALOOPER receives infos from the native_glue_fifo
+            android.LOGD("ALOOPER_POLL_WAKE")
```

Comment:
I'd rather it pushed an actual meaningful event, to keep the nastiness confined here instead of the extra random branch in front's input.lua this requires ;).

The rest of the stuff should have all the necessary bits to push a meaningful input event (or a custom MSC one).